release/get_token.py

```diff
@@ -0,0 +1,63 @@
+"""
+Provides a way to get a github token from disk
+or from the calling environment.
+
+If no token is available then it prompts the user to create one
+and saves it to disk.
+"""
+from __future__ import print_function
+
+import os
+
+SAVED_TOKEN_PATH = os.path.expanduser("~/.edx-release")
+
+# Token when passing via environment variable
+EDX_RELEASE_GITHUB_TOKEN = "EDX_RELEASE_GITHUB_TOKEN"
```

Comment:
I see that kind of call below, but don't understand why this line would need to be here (other than helping someone who wants to know what the required env vars are).